romancal/assign_wcs/tests/test_wcs.py

```diff
@@ -73,3 +72,15 @@ def test_wcs(tmpdir, distortion, step):
     ra, dec = l2_wcs.meta.wcs(-5, 3)
     assert np.isnan(ra)
     assert np.isnan(dec)
+
```

Comment:
I'd add a test for bbox = None and in the sregion has nan's in it.

Reply:
Added a check for `meta.bounding_box` $\ne$ `None`.
We already check for NaNs in `S_REGION` in the `assign_wcs_step.update_s_region_keyword` method.